.github/workflows/continuous-integration.yml

```diff
@@ -89,13 +90,14 @@ jobs:
         uses: actions/cache@v2
         env:
           cache-name: cache-build-deps
+          sbt-version: 1.5.3
         with:
           path: |
             ~/.dodo
             ~/.ivy2/cache
             ~/.ivy2/local/com.twitter
             ~/.sbt
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ matrix.scala }}-${{ matrix.java }}
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ env.sbt-version }}-${{ matrix.scala }}-${{ matrix.java }}
```

Comment:
Or not -- looks to be working. I wonder if this is picking up the image version since GitHub includes sbt by default on the images. Hmm, no it [includes sbt 1.5.2](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md) but the action is definitely using `1.5.3`:

```
Run actions/cache@v2
Cache not found for input keys: Linux-build-cache-build-deps-1.5.3-2.12.8-1.8
```
https://pipelines.actions.githubusercontent.com/E5R5BYwlkBmulEEqn39evCUbrE7hxAQv5ZbNMTEs4uibc5TJxV/_apis/pipelines/1/runs/127/signedlogcontent/17?urlExpires=2021-06-08T19%3A26%3A17.0718821Z&urlSigningMethod=HMACV1&urlSignature=wqWaFDuuOr8iZJ%2FTnR7mCrw8FoimeekTzqkkQbpIOvQ%3D